testcases/OpTestEEH.py

```diff
@@ -137,31 +170,31 @@ def get_dic_of_pe_vs_addr(self):
     #
     def run_pe_4(self, addr, e, f, phb, pe, con):
         self.prepare_logs()
-        count_old = self.check_eeh_slot_resets()
+    #    count_old = self.check_eeh_slot_resets()
         rc = self.inject_error(addr, e, f, phb, pe)
         if rc != 0:
             print "Skipping verification as command failed"
-            return
+            return EEH_MISS
         # Give some time to EEH PCI Error recovery
-        tries = 60
-        for i in range(1,tries):
-            time.sleep(1)
-            count = self.check_eeh_slot_resets()
-            if int(count) > int(count_old):
-                print "PE Slot reset happenned successfully on pe: %s" % pe
-                break
-            else:
-                print "PE Slot reset pe: %s, not yet done, (%d/%d)" % (pe,i,tries)
+     #   tries = 60
+     #   for i in range(1,tries):
+     #       time.sleep(1)
+     #       count = self.check_eeh_slot_resets()
+     #       if int(count) > int(count_old):
+     #           print "PE Slot reset happenned successfully on pe: %s" % pe
+     #           break
+     #       else:
+     #           print "PE Slot reset pe: %s, not yet done, (%d/%d)" % (pe,i,tries)
         self.gather_logs()
         if not self.check_eeh_pe_recovery(pe):
             msg = "PE %s recovery failed" % pe
             print msg
             # Don't exit here, continue to test with other adapters
             #raise OpTestError(msg)
-            return False
+            return EEH_RECOVERY_FAIL
         else:
             print "PE %s recovery success" % pe
-        return True
+        return EEH_RECOVERY_PASS
```

Comment:
Would it be better to throw an exception here? That way, if we're in a loop doing tests across adapters, we can catch the applicable exception and do cleanup/FFDC.